File: patterns/c-patterns/pattern108.c
// Pattern 108: ...

//   *   *   *
// *   *   *   *
//   *   *   *
// *   *   *   *
//   *   *   *
// *   *   *   *
//   *   *   *

#include <stdio.h>

int main()
{
    int n = 7;
    int i, j;

    for (i = 1; i <= n; i++)
    {
        for (j = 1; j <= n; j++)
        {
            if ((i + j) % 2 == 1)
            {
                printf("* ");
            }
            else
            {
                printf("  ");
            }
        }
        printf("\n");
    }
    return 0;
}

File: patterns/python-patterns/pattern108.py
# Pattern 108: ...

#   *   *   *
# *   *   *   *
#   *   *   *
# *   *   *   *
#   *   *   *
# *   *   *   *
#   *   *   *

n = 7

for i in range(1, n + 1):
    for j in range(1, n + 1):
        if (i + j) % 2 == 1:
            print("* ", end="")
        else:
            print("  ", end="")
    print()
